recipes-halif/pwrmgr/pwrmgr-hal-raspberrypi4.bb

```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "PWRMGR HAL Implementation."
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=b1e01b26bacfc2232046c90a330332b3"
+
+PROVIDES += " virtual/vendor-pwrmgr-hal "
+RPROVIDES_${PN} += " virtual/vendor-pwrmgr-hal"
+
+SRC_URI = "git://github.com/Mallikarjunm6316/rdkvhal-power-manager-raspberrypi4.git;branch=develop;protocol=https"
+
+SRCREV = "d164a5fe0d67f1ce8c8fddaf631e561c1e744466"
```

Comment:
This should be f174dc2bec24db8d19a2b384c6dccfaaa530c775

Reply:
Now it's pointing to main repo, so removed the src-rev.